conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinOperations.scala

```diff
@@ -0,0 +1,89 @@
+package tech.cryptonomic.conseil.indexer.bitcoin
+
+import cats.effect.{Concurrent, Resource}
+import com.typesafe.scalalogging.LazyLogging
+import fs2.Stream
+import slickeffect.Transactor
+
+import tech.cryptonomic.conseil.common.config.Platforms.BitcoinBatchFetchConfiguration
+import tech.cryptonomic.conseil.common.rpc.RpcClient
+import tech.cryptonomic.conseil.common.bitcoin.BitcoinPersistence
+import tech.cryptonomic.conseil.common.bitcoin.rpc.BitcoinClient
+import tech.cryptonomic.conseil.indexer.config.{Custom, Depth, Everything, Newest}
+
+/**
+  * Bitcoin operations for Lorre.
+  *
+  * @param bitcoinClient Bitcoin client instance
+  * @param persistence Bitcoin persistence instance
+  * @param tx [[slickeffect.Transactor]] to perform a Slick operations on the database
+  */
+class BitcoinOperations[F[_]: Concurrent](
+    bitcoinClient: BitcoinClient[F],
+    persistence: BitcoinPersistence[F],
+    tx: Transactor[F],
+    batchConf: BitcoinBatchFetchConfiguration
+) extends LazyLogging {
+
+  /**
+    * Start Lorre with mode defined with [[Depth]].
+    *
+    * @param depth Can be: Newest, Everything or Custom
+    */
+  def loadBlocks(depth: Depth): Stream[F, Unit] =
+    Stream
+      .eval(tx.transact(persistence.getLatestIndexedBlock))
+      .zip(bitcoinClient.getBlockChainInfo)
+      .evalTap(_ => Concurrent[F].delay(logger.info(s"Start Lorre for Bitcoin")))
```

Comment:
Is this log done in the right place? I mean there is a place https://github.com/Cryptonomic/Conseil/pull/867/files#diff-03717b0004757d8738510e597ecaafeaR91 where I guess we should print it. Here, we are already indexing blocks (accounts will be probably in a different method), so we are not starting Lorre here. We are starting fetching blocks.

Reply:
done